src/libsyntax/ext/build.rs

```diff
@@ -135,7 +135,7 @@ pub trait AstBuilder {
 
     fn expr_usize(&self, span: Span, i: usize) -> P<ast::Expr>;
     fn expr_int(&self, sp: Span, i: isize) -> P<ast::Expr>;
-    fn expr_u8(&self, sp: Span, u: u8) -> P<ast::Expr>;
+    fn expr_u32(&self, sp: Span, u: u32) -> P<ast::Expr>;
```

Comment:
`expr_u8` is dead code and the assoc fns for the other numeric types are missing.  Would it still be common practice to leave `expr_u8` and create a `expr_u32`?